db/knexfile.js

```diff
@@ -2,11 +2,24 @@
 
 const AppConstants = require("../app-constants");
 
+
+const TEST_DATABASE_URL = AppConstants.DATABASE_URL.replace(/\/(\w*)$/, "/test-$1");
+
 module.exports = {
-  client: "postgresql",
-  connection: AppConstants.DATABASE_URL,
-  pool: {
-    min: 2,
-    max: 10,
+  dev: {
+    client: "postgresql",
+    connection: AppConstants.DATABASE_URL,
+    pool: {
+      min: 2,
+      max: 10,
+    },
+  },
+  tests: {
+    client: "postgresql",
+    connection: TEST_DATABASE_URL,
+    pool: {
+      min: 2,
+      max: 10,
+    },
```

Comment:
Do we need a `production` key here as well? Or does OPs need to do anything special to deploy now that we're using an additional level of config nesting here?

Reply:
Yes, I'll plan to add a production section too.